src/displayapp/screens/WatchFacePineTimeStyle.cpp

```diff
@@ -340,13 +347,22 @@ void WatchFacePineTimeStyle::SetBatteryIcon() {
 }
 
 void WatchFacePineTimeStyle::AlignIcons() {
-  if (notificationState.Get() && bleState.Get()) {
-    lv_obj_align(bleIcon, sidebar, LV_ALIGN_IN_TOP_MID, 8, 25);
-    lv_obj_align(notificationIcon, sidebar, LV_ALIGN_IN_TOP_MID, -8, 25);
-  } else if (notificationState.Get() && !bleState.Get()) {
+  if (notificationState.Get() && alarmEnabled.Get()) {
+    lv_obj_align(alarmIcon, sidebar, LV_ALIGN_IN_TOP_MID, 9, 26);
+    lv_obj_align(notificationIcon, sidebar, LV_ALIGN_IN_TOP_MID, -9, 25);
+  } else if (notificationState.Get() && !alarmEnabled.Get()) {
     lv_obj_align(notificationIcon, sidebar, LV_ALIGN_IN_TOP_MID, 0, 25);
   } else {
-    lv_obj_align(bleIcon, sidebar, LV_ALIGN_IN_TOP_MID, 0, 25);
+    lv_obj_align(alarmIcon, sidebar, LV_ALIGN_IN_TOP_MID, 0, 26);
+  }
+
+  if (bleState.Get()) {
```

Comment:
please move the bluetooth alignment up before the notification/alarm alignment, to make the code better match the actual layout